helix-view/src/theme.rs

```diff
@@ -164,6 +171,39 @@ fn parse_color(value: Value) -> Option<Color> {
     }
 }
 
+fn parse_modifier(value: Value) -> Option<Modifier> {
+    if let Value::String(s) = value {
+        match s.as_str() {
+            "bold" => Some(Modifier::BOLD),
+            "dim" => Some(Modifier::DIM),
+            "italic" => Some(Modifier::ITALIC),
+            "underlined" => Some(Modifier::UNDERLINED),
+            "slow_blink" => Some(Modifier::SLOW_BLINK),
+            "rapid_blink" => Some(Modifier::RAPID_BLINK),
+            "reversed" => Some(Modifier::REVERSED),
+            "hidden" => Some(Modifier::HIDDEN),
+            "crossed_out" => Some(Modifier::CROSSED_OUT),
+            _ => None,
```

Comment:
Shouldn't we do some logging on invalid stuff?

Reply:
I added some warn statements, but these won't be shown to the user unless they specifically go digging...

Do we have a way of neatly reporting errors to the user like that? Implementing it might be out of scope for this pull request

Reply:
Not right now.